extensions/hibernate-orm/deployment/src/test/java/io/quarkus/hibernate/orm/multiplepersistenceunits/MultiplePersistenceUnitsInconsistentStorageEnginesTest.java

```diff
@@ -30,4 +31,10 @@ public void testInvalidConfiguration() {
         // deployment exception should happen first
         Assertions.fail();
     }
+
+    /**
+     * This is just to have the dialect matching MySQL and trigger the MySQL + storage engines check.
+     */
+    public static class H2DialectWithMySQLInTheName extends H2Dialect {
```

Comment:
funny :)

Reply:
Yeah, I didn't want to end up with MySQL there :).